### PR TITLE
Fixes ReplaceCulture attribute to use current UI culture and make Culture and UICulture be supplied via constructor

### PR DESCRIPTION
**Pull request by kichalla**
@dougbu @NTaylorMullen 

Fixes: https://github.com/aspnet/Testing/issues/61 and https://github.com/aspnet/Testing/issues/126

### CODE REVIEW

**Comment by kichalla** [comment]
Just FYI...I did run MVC against these changes and build passed without any test failures

**Comment by NTaylorMullen** [comment]
Add a single superrr simple test to validate correctness in this project (even if it works in MVC).

**Comment by kichalla** [comment]
updated. @NTaylorMullen @dougbu

File: Testing.sln [hunk 1]
```diff
@@ -36,6 +36,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		global.json = global.json
 	EndProjectSection
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNet.Testing.Tests", "test\Microsoft.AspNet.Testing.Tests\Microsoft.AspNet.Testing.Tests.xproj", "{C9F758D6-17A3-4BF1-BCCD-972B1DA959FC}"
```
Comment (Eilon) [inline Testing.sln]: Beautiful :smile:

File: test/Microsoft.AspNet.Testing.Tests/project.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "version": "1.0.0-*",
+    "dependencies": {
+        "Microsoft.AspNet.Testing": "1.0.0-*",
+        "xunit.runner.aspnet": "2.0.0-aspnet-*"
+    },
+    "commands": {
+        "test": "xunit.runner.aspnet"
+    },
+    "frameworks": {
+        "dnx451": { }
```
Comment (Eilon) [inline test/Microsoft.AspNet.Testing.Tests/project.json]: Can do `dnxcore50` for this too, no?

**Comment by dougbu** [comment]
Either in this PR or as a follow up Task, should add similar tests for `CultureReplacer`. Such tests would have caught fact `CultureReplacer` and `[ReplaceCulture]` weren't aligned.

**Comment by dougbu** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

File: test/Microsoft.AspNet.Testing.Tests/project.json (hunk 2 above)
Comment (kichalla) [inline test/Microsoft.AspNet.Testing.Tests/project.json]: good catch, added it now. Updated

**Comment by Eilon** [comment]
:shipit:

**Comment by kichalla** [comment]
eb727006987e2dac556e1924e7610b26560562b5

**Pull request closed without merging**